app/assets/javascripts/bastion/components/notification.service.js

```diff
@@ -16,8 +16,15 @@ angular.module('Bastion.components').service("Notification", ['$interpolate', 'f
         return result;
     }
 
-    this.setSuccessMessage = function (message, context) {
-        foreman.toastNotifications.notify({message: interpolateIfNeeded(message, context), type: 'success'});
+    this.setSuccessMessage = function (message, options) {
+        /* eslint-disable no-unused-expressions, angular/ng_definedundefined*/
+        var baseOptions = {};
```

Comment:
You can initialize a variable and then assign a value to it later. This is a good practice in JS (well, for es5) because of [hoisting](https://scotch.io/tutorials/understanding-hoisting-in-javascript), which is why eslint recommends it.

i.e.
```javascript
var baseOptions, fullOptions;
// other stuff
baseOptions = { ham: "sandwich" };
fullOptions = { turkey: 'club' };